ouroboros-consensus-byron-test/src/Ouroboros/Consensus/ByronDual/Node.hs

```diff
@@ -222,7 +222,7 @@ protocolInfoDualByron abstractGenesis@ByronSpecGenesis{..} params credss =
 instance NodeInitStorage DualByronBlock where
   -- Just like Byron, we need to start with an EBB
   nodeInitChainDB cfg InitChainDB { getCurrentLedger, addBlock } = do
-      tip <- ledgerTipPoint (Proxy @DualByronBlock) <$> getCurrentLedger
+      tip <- ledgerTipPoint <$> getCurrentLedger
```

Comment:
Question: How is this change and similar other changes related to that PR purpose?